Frameworks/Foundation/NSSet.mm

```diff
@@ -526,6 +526,9 @@ - (NSArray*)sortedArrayUsingDescriptors:(NSArray*)descriptors {
  @Status Interoperable
 */
 - (id)member:(id)obj {
+    if (obj == nil) {
+        return nil;
+    }
```

Comment:
discussed offline. we still want this check and ideally would like a nil check on adding a member in the mutable version. The change, as is, is good though.

---
In reply to: [63953430](https://github.com/Microsoft/WinObjC/pull/499#discussion_r63953430) [](ancestors = 63953430)